src/ui.rs

```diff
@@ -1197,6 +1212,14 @@ impl PofToolsGui {
                         }
                     }
                 }
+                Warning::InvertedBBox(id_opt) => {
+                    if let Some(id) = id_opt {
+                        model.sub_objects[id].bbox.is_inverted()
```

Comment:
What if the bbox is `BBox::EMPTY` because we recalculated it as such? Should those be sanitized or should the warning skip it?

Reply:
Good point, I would say to not warn, because it is unlikely to be accidental but sanitize on write